recipes/LibriSpeech/ASR/transformer/hparams/branchformer_large.yaml

```diff
@@ -255,10 +256,10 @@ augmentation: !new:speechbrain.lobes.augment.SpecAugment
     freq_mask: True
     n_freq_mask: 2
     time_mask: True
-    n_time_mask: 3
+    n_time_mask: 10
```

Comment:
this is interesting, that is a lot of masks. It is fine I guess because LibriSpeech mean utternace length is like around 5-6 seconds IRCC. 
A better approach would be variable length SpecAugment. 
So the main issue was overfitting. Label smoothing and more specaugment fixed it.

Reply:
I have an implementation for variable length specaugm on local. I am too lazy for a PR. Do you want to do it ?

Reply:
This one I don't know, maybe it's too much... I'd love a variable spec augment. Could you PR and we update?